olp-cpp-sdk-core/src/client/OlpClient.cpp

```diff
@@ -342,13 +363,24 @@ HttpResponse OlpClient::CallApi(
     std::string path, std::string method,
     std::multimap<std::string, std::string> query_params,
     std::multimap<std::string, std::string> header_params,
-    std::multimap<std::string, std::string> forms_params,
+    std::multimap<std::string, std::string> form_params,
+    std::shared_ptr<std::vector<unsigned char>> post_body,
+    std::string content_type, CancellationContext context) const {
+  auto http_verb = GetHttpVerb(method);
+  return CallApi(path, http_verb, query_params, header_params, form_params, post_body, content_type, context);
```

Comment:
```suggestion
  return CallApi(path, http_verb, query_params, header_params, form_params,
                 post_body, content_type, context);
```
Clang format please. Exceeds max line width